src/index.js

```diff
@@ -19,21 +19,24 @@ import Round from './Round';
 import Turn from './Turn';
 import domUpdates from './domUpdates.js';
 
-var data;
+var boards;
 fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
-  .then(function(response) {
-    return response.json()
-  })
-  .then(function(parsedData) {
-    data = parsedData.data
-    console.log(data)
-  })
+  .then(response => response.json())
+  .then(parsedData => getData(parsedData))
```

Comment:
`getData` is kind of a weird function name here, you just got the data with that fetch call — it looks like your getData function is actually doing something more like `generateClue`. Also a little odd that we’re passing through *all* the data here, but then only generating a single `Clue` based on that data? I’d expect that a single `Clue` instance only represents a single clue object, not the entire dataset. If your `Clue` class does need all the fetched data, I’d potentially rename it to `Clues`